pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -653,6 +653,7 @@ func TestFitScore(t *testing.T) {
 		existingPods         []*v1.Pod
 		expectedPriorities   framework.NodeScoreList
 		nodeResourcesFitArgs config.NodeResourcesFitArgs
+		disablePreScore      bool
```

Comment:
in general, avoid using negative wording in variable names. You can say `enablePreScore` 
 or `runPreScore` instead.

Reply:
> in general, avoid using negative wording in variable names. You can say `enablePreScore` or `runPreScore` instead.

But I think some other palce like `NodeAffinity` or `Prefilter` are using `disablePreScore` or `disablePreFilter`, should we keep the consistent?

Reply:
> use `enable` here and we can change the others in a follow up

Ok, I'd like to do this in follow PR.